pkg/controller/controller.go

```diff
@@ -122,7 +122,7 @@ func (c *controller) Run(workers int, stopCh <-chan struct{}) error {
 	}
 
 	<-stopCh
-	log.Info("shutting down queue as workqueue signaled shutdown")
+	log.V(logf.WarnLevel).Info("shutting down queue as workqueue signaled shutdown")
```

Comment:
I would say this should be info rather than warn.

Reply:
good point! fixed